includes/ets_admin_qa_function.php

```diff
@@ -411,7 +411,10 @@ public function admin_qa_save( $productId ){
 				"user_email"      =>   $newUserEmail,
 				"user_id"	      =>   $newUserId, 
 				"approve"		  =>   $admin_approve 
-			); 
+			);
+
+			$productFirstQa = apply_filters('ets_new_first_question', $productFirstQa, $productId);
```

Comment:
There is no need to have a separate filter for the first question, it should be ets_new_question here as well, so that developers of our plugin do not have to remember this separate hook for first question, which is a fault in our design.